qemu/tests/virtio_fs_share_data.py

```diff
@@ -126,8 +126,7 @@ def get_stdev(file):
     # st_dev check config
     cmd_get_stdev = params.get("cmd_get_stdev")
     nfs_mount_dst_name = params.get("nfs_mount_dst_name")
-
-    if cmd_xfstest:
+    if cmd_xfstest and not params.get("setup_hugepages"):
```

Comment:
this `not params.get("setup_hugepages"):` might not right.
if the `setup_hugepages = yes` and `setup_hugepages = no` is defined in cfg, this if block will awalys run.